src/components/Agreements/ViewAgreement/Sections/Organizations.js

```diff
@@ -24,32 +25,64 @@ export default class Organizations extends React.Component {
     }).isRequired,
     id: PropTypes.string,
     onToggle: PropTypes.func,
+    organizations: PropTypes.object,
     open: PropTypes.bool,
   };
 
-  renderOrgList = (orgs) => {
+  renderOrgList = () => {
+    const { organizations } = this.props;
     return (
       <React.Fragment>
-        {orgs.map(o => (
-          o.org ?
-            <Layout data-test-organizations-org className="marginTopHalf" key={`${o.org.id}-${o.role.value}`}>
-              {o.org.orgsUuid ?
-                <Link to={`/organizations/view/${o.org.orgsUuid}`}>{o.org.name}</Link> :
-                o.org.name
-              }
-              {o.role && `, ${o.role.label}`}
-            </Layout> : null
-        ))}
+        {organizations && organizations.map(o => {
+          const interfaces = get(o.org, ['orgsUuid_object', 'interfaces'], []);
+          return (
+            o.org ?
+              <Card
+                data-test-organizations-org
+                key={`${o.org.id}-${o.role.value}`}
+                header={
+                  <span className="my-card-header">
+                    {
+                      o.org.orgsUuid ?
+                        <Link to={`/organizations/view/${o.org.orgsUuid}`}>
+                          {o.org.name}
+                          .
```

Comment:
The mockup for this uses an [interpunct](https://en.wikipedia.org/wiki/Interpunct) with space around it, not a period.

Reply:
Also, the role is not part of the link in the mockup.